README.md

```diff
@@ -1,95 +1,209 @@
 # Hugging Face Llama Recipes
 
-🤗🦙Welcome! This repository contains minimal recipes to get started with Llama 3.1 quickly.
+![thumbnail for repository](./assets/hf-llama-recepies.png)
 
-* To get an overview of Llama 3.1, please visit [Hugging Face announcement blog post](https://huggingface.co/blog/llama31).
+🤗🦙Welcome! This repository contains *minimal* recipes to get started quickly
+with **Llama 3.x** models, including **Llama 3.1** and **Llama 3.2**.
+
+* To get an overview of Llama 3.1, please visit [Hugging Face announcement blog post (3.1)](https://huggingface.co/blog/llama31).
+* To get an overview of Llama 3.2, please visit [Hugging Face announcement blog post (3.2)](https://huggingface.co/blog/llama32).
 * For more advanced end-to-end use cases with open ML, please visit the [Open Source AI Cookbook](https://huggingface.co/learn/cookbook/index).
 
 This repository is WIP so that you might see considerable changes in the coming days.
 
-_Note: To use Llama 3.1, you need to accept the license and request permission to access the models. Please, visit [any of the Hugging Face repos](https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct) and submit your request. You only need to do this once, you'll get access to all the repos if your request is approved._
+> [!NOTE]
+> To use Llama 3.x, you need to accept the license and request permission
+to access the models. Please, visit [any of the Hugging Face repos](https://huggingface.co/meta-llama)
+and submit your request. You only need to do this once, you'll get access to
+all the repos if your request is approved.
+
+## Getting Started
+
+The easiest way to quickly run a Llama 🦙 on your machine would be with the
+🤗 `transformers` repository. Make sure you have the latest release installed.
+
+```shell
+$ pip install -U transformers
+```
+
+### Generate text with a base model
+
+Let's generate some text from a given prompt.
+
+```python
```

Comment:
Can we use `pipeline` here? It would simplify the code snippets a lot which is important for the main README (no need for people to think about tokenizers, torch, generation params, etc)